packages/donate-button-v4/src/components/widget/components/Faq/index.tsx

```diff
@@ -26,10 +24,16 @@ interface FaqItemTypes {
 	title: string;
 	description: VNode;
 	mobileOnly?: boolean;
+	hideSlugList?: string[];
 }
 
 const FaqItem: FunctionalComponent<{faqData: FaqItemTypes}> = ({faqData}) => {
 	const [isOpen, setOpen] = useState(false);
+	const {nonprofitSlug} = useConfigContext();
+
+	if (faqData.hideSlugList?.includes(nonprofitSlug)) {
```

Comment:
Do we want this section to **NOT** show to IRC only?

Reply:
Yep, their request! Is there a synonym for `hideSlugList` that would work better?